app/controllers/country_bands_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
 
   def edit
     @number = id_from_params
-    unless CountryBand::BANDS.keys.include?(@number)
+    unless CountryBandDetail.distinct.pluck(:number).include?(@number)
```

Comment:
In case of CountryBand, where there's only like 5 or so entries the whole time, loading all values from the DB into memory won't be a big problem. But in general, loading the whole column into memory just for a simple `include?` check is a _big_ problem, so let me show you the cool SQL way
```suggestion
    unless CountryBandDetail.exists?(number: @number)
```

Reply:
This (roughly, not sure how Rails does internal optimizations exactly) translates to a `SELECT 1 FROM country_bands WHERE number = 3` query